blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+have been accepted by the SIP Committee and are under development in the compiler.
+
+### Documentation and MOOCs
+
+**We improved the usability of the Scala website (this website).** The website had not seen a major rework since 2013,
+and it does not represent the full picture that we would like Scala newcomers to see. To this effect, we have removed
+the split of Scala 2 vs Scala 3 on [the landing page](https://github.com/scala/scala-lang/pull/1344) and on
+[the documentation](https://github.com/scala/docs.scala-lang/pull/2430), consolidated the “Getting Started” instructions
+([scala-lang#1344](https://github.com/scala/scala-lang/pull/1344),
+[scala-lang#1348](https://github.com/scala/scala-lang/pull/1348),
+[scala-lang#1352](https://github.com/scala/scala-lang/pull/1352),
+[scala-lang#1373](https://github.com/scala/scala-lang/pull/1373),
+[docs.scala-lang#2388](https://github.com/scala/docs.scala-lang/pull/2388),
+[docs.scala-lang#2520](https://github.com/scala/docs.scala-lang/pull/2520)), improved the
+[contributing guide](https://github.com/scala/docs.scala-lang/pull/2350), and created an infrastructure to show both
+Scala 2 and Scala 3 versions of code examples in the documentation
+([docs.scala-lang#2450](https://github.com/scala/docs.scala-lang/pull/2450)). We created an
+[issue](https://github.com/scala/docs.scala-lang/issues/2481) to coordinate the update of the whole documentation to
+always show both Scala 2 and Scala 3 code examples when possible.
+
+**We offered individualized support to our MOOCs’ learners.** We published the course “Effective Programming in Scala”
+to the [Extension School platform](https://www.extensionschool.ch/learn/effective-programming-in-scala). This
+platform allows the learners to have regular 1-on-1 meetings with our instructors along their learning journey. Learn
+more about the motivation in the previous [announcement]({% link blog/_posts/2022-06-08-learn-scala-at-epfl-extension-school.md %}).
+
+### Developer Experience
+
+**We improved the debugger for Scala 3.** The Metals’ debugger is now able to evaluate arbitrary Scala 3 expressions
+while the debugger is paused in the middle of a program. Step-by-step execution now automatically skips the methods
+generated by the compiler (e.g. mix-in forwarders, getters, setters, bridge methods, and synthetic methods of case
+classes). Learn more in [this video](https://www.youtube.com/watch?v=UePrOa_1Am8).
+
+**We fixed the apps installed by the [setup instructions](https://docs.scala-lang.org/getting-started).** The
+command-line apps `sbt` and `scala` now behave consistently regardless of the installation process (`cs install` vs
+manual download of the release artifacts).
+
+### Community and Contributor Experience
+
+**We co-organized the [ScalaCon](http://www.scalacon.org/) online conference.** We were in charge of the program and
+the program committee.
```

Comment:
suggest: "The talk videos are available online here (link)"

The URL is http rather than https; is that intentional?

Reply:
Done in 6c333175